app/views/challenges/_header.html.erb

```diff
@@ -1,7 +1,7 @@
 <div id='challenge-header' class='<%= controller_name %>'>
   <div class="container">
     <div class="row">
-      <div class="col-sm-10 col-sm-offset-1">
+      <div class="col-md-10 col-md-offset-1 col-sm-12 padding-horizontal-none">
```

Comment:
It can cause some pretty nasty issues to add overrides to padding on the left/right of row / col classes, so I'd avoid doing this if possible. Any thoughts / ideas as to what you're hoping to do here?